jenkins/custom-ci/generic/custom-ci.groovy

```diff
@@ -185,11 +191,12 @@ pipeline {
                         script { build_stage = env.STAGE_NAME }
                         script {
                             try {
-                                def rgwintegrationbuild = build job: '/GitHub-custom-ci-builds/generic/cortx-rgw-integration-build', wait: true,
+                                def rgwintegrationbuild = build job: '/Release_Engineering/re-workspace/AbhirupSpace/cortx-rgw-integration', wait: true,
```

Comment:
change job to original integration build instead of your test job.

Reply:
After testing i will revert back to original. Its for testing .

Reply:
Done